### PR TITLE
12.0 add sale order no create partner

### PR DESCRIPTION
**Pull request by MVillaescusaM**

### CODE REVIEW

**Review by angelmoya — APPROVED** [review]
tested on runbot :+1:

**Review by atchuthan — APPROVED** [review]

File: sale_order_no_create_partner/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+# # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+{
+    'name': "Sale Order no create partner",
+    'summary': "Avoid creating a new partner in the sales order section",
+    'version': '12.0.1.0.0',
+    'category': 'Sales',
+    'website': 'https://github.com/OCA/',
+    'author': 'PESOL, Odoo Community Association (OCA)',
+    'maintainers': 'MarioVillaescusaPesol',
```
Comment (rousseldenis) [inline sale_order_no_create_partner/__manifest__.py]: This should be in a list.

File: sale_order_no_create_partner/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,19 @@
+# Copyright 2020 PESOL>
+# # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+{
+    'name': "Sale Order no create partner",
+    'summary': "Avoid creating a new partner in the sales order section",
+    'version': '12.0.1.0.0',
+    'category': 'Sales',
+    'website': 'https://github.com/OCA/',
```
Comment (rousseldenis) [inline sale_order_no_create_partner/__manifest__.py]: You should point the repo. 
```suggestion
    'website': 'https://github.com/OCA/sale-workflow',
```

File: sale_order_no_create_partner/__manifest__.py [hunk 3]
```diff
@@ -0,0 +1,19 @@
+# Copyright 2020 PESOL>
+# # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
```
Comment (rousseldenis) [inline sale_order_no_create_partner/__manifest__.py]: Please set url to https

File: sale_order_no_create_partner/__init__.py [hunk 4]
```diff
@@ -0,0 +1 @@
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
```
Comment (rousseldenis) [inline sale_order_no_create_partner/__init__.py]: You can remove license here. Not necessary

**Review by rousseldenis — CHANGES_REQUESTED** [review]
Code review. Some changes

**Review by HaraldPanten — APPROVED** [review]

**Comment by MVillaescusaM** [comment]
@rousseldenis Changes done, can you approve the changes?

**Comment by rousseldenis** [comment]
@MarioVillaescusaPesol

**Comment by angelmoya** [comment]
Hi @rousseldenis ... I'm not sure, can I add permission to OCA-git-bot to our repository... this way this will work

**Comment by rousseldenis** [comment]
@angelmoya No need to. Could you rebase ?

**Comment by angelmoya** [comment]
Ok, rebase done

**Pull request closed without merging**